docs/examples/set_app_element.md

```diff
@@ -1,7 +0,0 @@
-# Using setAppElement
```

Comment:
Why this was removed?

Reply:
Maybe I'm missing historical context but I removed it because it feels more of a requirement than a suggestion since a large warning is logged to the console when it's omitted and since every other example already included it. But if it's needed, then sure, I'll add it back in just a few.

Reply:
Maybe the explaination is bad. `setAppElement` is a feature that is misused all the time.
Problem is that users always place the setAppElement together with the modal utilization, 
which causes problems when testing and others (i'll try to reference them).
Suggestions are welcome.

Reply:
OK, added back in. No strong suggestions, I just assumed `setAppElement` wasn't as strongly recommended in the past or something and might not be needed today.

Reply:
Actually `setAppElement` must be used. Historically the default element that react-modal uses to add the `aria-hidden` attribute is wrong. By default it will use the `document.body` which will turn the whole page inaccessible.